gallery_dl/extractor/vsco.py

```diff
@@ -79,7 +79,7 @@ def images(self):
     def _extract_preload_state(self, url):
         page = self.request(url, notfound=self.subcategory).text
         return util.json_loads(text.extr(page, "__PRELOADED_STATE__ = ", "<")
-                               .replace('"prevPageToken":undefined,', ''))
+                               .replace("undefined", "null"))
```

Comment:
This is too general and would replace all occurrences of `undefined`, even inside a string value, with `null`. 
```suggestion
                               .replace('":undefined', '":null'))
```

Reply:
True, didn't consider that